docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -135,6 +135,75 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/wired-status-online-connect-automatically-selected.png" %}
 
+
+*   {: tab="Debian_commandline"}{% include /style/icon.html type="info" %}
+    These instructions were written using Debian look-alikes for those wanting to know what happens under the hood.
+    The commands need to be executed by or as root.
+    {:class="alert alert-info"}
+
+    1.  Connect the USB cable to your computer and its mini-B plug to the EV3 (slot marked "PC").
+
+    2.  Verify that USB tethering is disabled: "Wireless and Networks" > "Tethering.
+
+    3.  Set default Linux network settings: "Wireless and Networks" > "All Network Connections" > "Wired" >
+        "IPv4" > "Change" > "Load Linux Defaults".  The EV3 should get address 10.42.0.3, gateway 10.42.0.1.
+
+    4.  Verify if the EV3 is recognized by your Linux PC:  "lsusb" should show
+        
+        `Bus 001 Device 002: ID 0525:a4aa Netchip Technology, Inc. Linux-USB CDC Composite Gadge (Ethernet and ACM)`
```

Comment:
> In lists, it is still indented 4 extra spaces from the list text, so it looks exactly the same as normal text to me.

It's extremely difficult to differentiate between sub-bullets, indented additional lines, and code blocks; they all are indented in the same way. Using indentation for code blocks is an unnatural use of whitespace, and as a result it conflicts with all the _other_ uses of whitespace.

> And regardless of any opinions on which is "better", the rest of the website uses indents, so I would like to stay consistent with that.

_argumentum ad antiquitatem_. "It has always been done this way" isn't a reason to _continue_ doing it; switching to code fences is trivial and something that I would happily put in the work for now that we have that ability.

> How so?

Because you must maintain the indentation. Every line must be prepended with four spaces, so not only do you need to add the spacing before every line, but you also need to mess with indentation when copy-pasting code. This is especially difficult in GitHub's online editor.